integration-tests/tests/utils/config.py

```diff
@@ -9,6 +9,16 @@ class PackageConfig:
     clp_package_dir: Path
 
     def __post_init__(self):
+        # Check if package directory exists and is valid
```

Comment:
sorry for my over sights.
since we are reusing "self.clp_package_dir" multiple times, let's do something like
```
package_dir = self.clp_package_dir
```
and reuse `package_dir ` instead.